### PR TITLE
added back to top fab

### PR DESCRIPTION
**Pull request by FazeenIjaz**
Add a FAB to scroll back to top.

### CODE REVIEW

**Comment by aviramha** [comment]
Thanks for sending this. Can you remove unrelated changes (css, lock files) and leave only needed changes so it'll be easier to review?

File: layouts/index.html [hunk 1]
```diff
@@ -1,12 +1,51 @@
...
 <main id="content" class="site-content">
-    {{- partial "sections/section-hero.html" . }}
-    {{- partial "sections/section-cli.html" . }}
-    {{- partial "sections/section-ide.html" . }}
-    {{- partial "sections/section-quick-start-link.html" . }}
-    {{- partial "sections/section-use-cases.html" . }}
-    {{- partial "sections/section-all-stories.html" . }}
-    {{- partial "sections/section-how-it-works.html" . }}
-    {{- partial "sections/section-get-support.html" . }}
+  {{- partial "sections/section-hero.html" . }} {{- partial
```
Comment (aviramha) [inline layouts/index.html:3]: Change back to previous formatting.

File: assets/css/site.css
Comment (aviramha) [inline assets/css/site.css:1]: Remove from PR

File: package-lock.json
Comment (aviramha) [inline package-lock.json:1]: Remove from PR

File: yarn.lock
Comment (aviramha) [inline yarn.lock:1]: Remove from PR

**Review by aviramha — CHANGES_REQUESTED** [review]
Thanks, still some left over changes that needs to rollback.

**Comment by aviramha** [comment]
Nothing from your side. I'm waiting to run it locally to see that it works as intended before merigng.

**Comment by infiniteregrets** [comment]
Hey @FazeenIjaz! Thanks for making this change. I ran your branch and found that this button does not show up on the documentation page or any other page. Could you please look into that if possible?

Also, the button is really small and looks a bit odd per the rest of the UI. Can you please try to resize the button a bit, move it a little to the left and add some padding 🙂 

Screenshot 2024-02-07 at 9 06 51 AM

**Comment by infiniteregrets** [comment]
Hi @FazeenIjaz I have made some changes on my end here to adjust the UI. I appreciate you implementing this! Thanks for your contribution (:

**Review by infiniteregrets — APPROVED** [review]

**Review by aviramha — APPROVED** [review]

**Pull request merged**